query-service-impl/src/main/java/org/hypertrace/core/query/service/trino/converters/DefaultColumnRequestConverter.java

```diff
@@ -111,26 +111,35 @@ private String convertFilterToString(
               convertExpressionToString(filter.getRhs(), paramsBuilder, trinoExecutionContext));
           break;
         case CONTAINS_KEY:
+          builder.append(operator);
```

Comment:
```suggestion
          builder.append(MAP_FUNCTION);
```

Reply:
if we treat `element_at` as function.

Reply:
`operator` is initialised by `convertOperatorToString()` in a generic way, which will return `MAP_FUNCTION` in this case